cql/CollateManagementData.cql

```diff
@@ -238,6 +242,12 @@ define MostRecentCytologyReport:
   else
     null
 
+define SecondMostRecentCytologyReport:
+  if Count(SortedCytologyReports) > 0 then
```

Comment:
```suggestion
  if Count(SortedCytologyReports) > 1 then
```
We probably only want to query the array for the second most recent cytology report if we know that two exist.